debian/changelog

```diff
@@ -1,3 +1,9 @@
+containerd.io (1.6.31-2) release; urgency=medium
+
+  * update containerd binary to v1.6.31.
```

Comment:
We should build with latest go1.21 version (upstream didn't update yet, but if we build our packages, we can override the version)